api/src/test/java/io/grpc/ManagedChannelRegistryAccessor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 The gRPC Authors
+ * Copyright 2019 The gRPC Authors
```

Comment:
2020? Also similar to my other comment should this be called `ManagedChannelProviderRegistryAccessor`?